test/unittests/api_test.cpp

```diff
@@ -44,10 +44,10 @@ TEST(api, execution_result_void)
 
 TEST(api, execution_result_value)
 {
-    const ExecutionResult result = Value{1234};
+    const ExecutionResult result = Value{1234_u32};
```

Comment:
If this i32 now, shouldn't there be another test for i64/f32/f64 (pick one or more?)?

Reply:
Not sure what this test is about. Seems to testing implicit Value -> ExecutionResult conversion.

Reply:
@gumb0 ?

Reply:
Yeah this tests `ExecutionResult` constructor taking `Value`

Reply:
> Yeah this tests `ExecutionResult` constructor taking `Value`

Right, I guess implicit one, because there's also a test for explicit constuction below.